File: Apna College(c++)/Lecture_08/reverseArray.cpp
#include<iostream>
using namespace std;

void revArray(int arr[],int sz){
    int i=0,j=sz-1;
    while(i<j){
        int temp = arr[i];
        arr[i]=arr[j];
        arr[j]=temp;
        i++;
        j--;
    }
}

int main(){
    int arr[] = {2,4,5,6,7,13,25,33,55};
    int sz = sizeof(arr)/sizeof(int);
    revArray(arr,sz);
    for(int i=0;i<sz;i++){
        cout << arr[i] << " ";
    }
    return 0;
}

File: GFG (Java)/06_Arrays/src/ReverseArray.java
public class ReverseArray {
    public static void main(String[] args) {
        int arr[] = {2,6,99,56,34,67,23,99};
        int i=2;
        int j=i+3;
        while(i<j){
            int temp = arr[i];
            arr[i++]= arr[j];
            arr[j--] = temp;
        }
        for(int num: arr){
            System.out.print(num+" ");
        }
    }
}
